hathor/transaction/storage/transaction_storage.py

```diff
@@ -50,6 +50,18 @@ class TransactionStorage(ABC):
 
     log = get_logger()
 
+    # Key storage attribute to save if the network stored is the expected network
+    _network_attribute: str = 'network'
+
+    # Key storage attribute to save if the full node is running a full verification
+    _running_full_verification_attribute: str = 'running_full_verification'
+
+    # Key storage attribute to save if the manager is running
+    _manager_running_attribute: str = 'manager_running'
+
+    # Ket storage attribute to save the last time the node started
+    _last_start_attribute: str = 'last_start'
```

Comment:
Maybe rename to `db_last_start`?

Reply:
But it's also the last time the node itself started. I don't think there's any difference between the "node" and the "node's database", so this can also be used for logging when was the last time the node started.